docs/ref/django-admin.txt

```diff
@@ -1793,15 +1793,48 @@ Extra niceties
 Syntax coloring
 ---------------
 
+.. versionchanged:: 3.1
+
+  Added support for colored terminal output via colorama_.
+
 The ``django-admin`` / ``manage.py`` commands will use pretty
 color-coded output if your terminal supports ANSI-colored output. It
 won't use the color codes if you're piping the command's output to
 another program unless the :option:`--force-color` option is used.
 
-Under Windows, the native console doesn't support ANSI escape sequences so by
-default there is no color output. But you can install the `ANSICON`_
-third-party tool, the Django commands will detect its presence and will make
-use of its services to color output just like on Unix-based platforms.
+Under Windows, the legacy ``CMD.exe`` native console doesn't support ANSI
+escape sequences so by default there is no color output. But there are three
+ways to get color output on Windows from Django:
+
+* Install `ANSICON`_, a third-party tool that allows ``CMD.exe`` to process
+  ANSI color codes. Django commands will detect its presence and will make use
+  of its services to color output just like on Unix-based platforms.
+* Use a newer version of PowerShell or `Windows Terminal
+  <https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701>`_
```

Comment:
So, this trick works... 

Can we determine if colors are available? (Can we detect the new terminal?)